server/data/entryData.js

```diff
@@ -10,64 +10,64 @@ const entryData = [
   },
   {
     "user_id": 1,
     "date": "2023-04-01",
-    "feeling": "jealous",
-    "activity": "Get over it.",    
-    "journal_entry": "Today I am feeling anxious because I pooped my pants in front of my crush."
+    "feeling": "bad",
+    "activity": "Create a cozy blanket fort with lots of pillows, and take a nap or watch your favorite movie.",    
+    "journal_entry": "My best friend found out that they're adopted."
   },
   {
     "user_id": 1,
     "date": "2023-04-08",
     "feeling": "friendly",
-    "activity": "Kiss a stranger on the mouth.",
-    "journal_entry": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+    "activity": "Put on a silly outfit, your favorite playlist, and perform the weirdest dance moves you can conjure.",
+    "journal_entry": "I met a new person at my coworking space who is really nice. We're meeting for coffee later this week!"
   },
   {
     "user_id": 1,
     "date": "2023-04-09",
     "feeling": "cheerful",
     "activity": "Put on an irresistible song and wiggle your body the weirdest way you can. Decide on the movements based on listening to how your body wants to move, stretch, and shake out, NOT on if the dance moves are good. The more like Ace Ventura: Pet Detective meets hippie at Woodstock the better", 
-    "journal_entry": "Today I am feeling anxious because I pooped my pants in front of my crush."
+    "journal_entry": "I'M ON TOP OF THE WORLD!!!"
   },
   {
     "user_id": 1,
     "date": "2023-04-10",
-    "feeling": "happy",
-    "activity": "Be more sad.",
-    "journal_entry": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+    "feeling": "bad",
+    "activity": "Create a whimsical talisman: Craft a silly and unusual object or trinket that represents your bad mood. This can be anything from a miniature rubber chicken to a glittery rock. Keep it with you as a reminder to take things less seriously and embrace the playful side of life.",
+    "journal_entry": "I wasn't feeling great, but then I created a whimsical talisman and felt much better! I edited this entry after :) Ever seen Hereditary? My talisman was similar to the dolls that 'little girl' made."
```

Comment:
😨 I HAVE seen Hereditary and now I'm scared of our user........  Anyway, I love these updates, Joe. Such curious and impactful storytelling in just a few sentences.